R/pkg/R/generics.R

```diff
@@ -1046,8 +1084,8 @@ setGeneric("ntile", function(x) { standardGeneric("ntile") })
 #' @export
 setGeneric("n_distinct", function(x, ...) { standardGeneric("n_distinct") })
 
-#' @rdname percent_rank
-#' @export
+# @rdname percent_rank
```

Comment:
Yeah that works if we also set the argument list of the generic counterpart to `...`. Then we still have to document the `...` part. We could tell the user things like "this should be left empty". I'm not sure if that would cause some confusion?